packages/nestjs/package.json

```diff
@@ -50,8 +50,8 @@
     "@sentry/utils": "8.36.0"
   },
   "devDependencies": {
-    "@nestjs/common": "^8.0.0 || ^9.0.0 || ^10.0.0",
```

Comment:
IMHO no need to define this like this, this should define the actual version we want to use locally.